scan_matching:
  ros__parameters:
    global_frame_id: "map"
    robot_frame_id: "base_link"
    lidar_frame_id: "front_lidar"
    registration_method: "NDT"
    ndt_resolution: 1.0
    ndt_num_threads: 5
    gicp_corr_dist_threshold: 5.0
    vg_size: 1.0
    R_variance: 6.0
    use_ekf: true
    neglect_roll_pitch_z: true

